pages/glossary/distinguishable-styles.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Distinguishable Styles
+key: distinguishable-styles
+unambiguous: true
+objective: true
+---
+
+Two elements have _distinguishable styles_ if at least one of the following is true:
+
+- they have [distinguishable colors](#distinguishable-colors); or
+- they have [distinguishable borders](#distinguishable-borders); or
+- they have [distinguishable box-shadows](#distinguishable-box-shadows); or
+- they have different [computed values][] for the `background-image` property; or
+- they have different [computed values][] for at least one [text style property](#text-style-properties).
```

Comment:
There's an issue with this case, which also applies to the `border` case: If the surrounding text has an underline, but the link doesn't, the link will still appear with an underline and therefore won't be distinguishable from the text. Consider this:

```html
<style>
  p {
    text-decoration: underline;
  }

  a {
    text-decoration: none;
  }
</style>

<p>Hello <a href="#">link</a></p>
```

The above renders as:

![image](https://user-images.githubusercontent.com/1174718/98812371-ac1cf900-2422-11eb-9ec6-89280d0f7054.png)

This case will however pass as the computed `text-decoration` properties of the two elements are different.

Reply:
This seems to apply to `background-image` as well; if the parent has a background image, but the link doesn't, that won't make the link distinguishable.

Reply:
These are actually not too big problems given that they "only" make the rule pass too often and "only" generate false negatives.

Reply:
Let me try to have a go at this.

Reply:
@kasperisager and @Jym77 can you guys have a look at the new definitions?